src/components/DeckMechanisms/index.js

```diff
@@ -291,14 +288,14 @@ export default class DeckMechanisms extends React.Component {
         // See: https://discordapp.com/channels/293674725069029377/564840207875178502/676580933180325920
         // Note: it seems that QoH spawns do not cause a weighing of the deck.
         // See: https://discordapp.com/channels/293674725069029377/564840207875178502/676580198057246730
-        satyr1 = arrayRandom(satyrs)
+        satyr1 = arrayRandom(satyrs).id
```

Comment:
The play function accepts an id - this would cause an error when playing Queen of Herds